File: src/TaskManager.WebApi/Models/CronTask.cs
﻿namespace TaskManager.WebApi.Models
{
    public class CronTask
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Minutes { get; set; }
        public string Hours { get; set; }
        public string Days { get; set; }
        public string Months { get; set; }
        public string Weekdays { get; set; }
        public int UserId { get; set; }
        public int ApiId { get; set; }
    }
}

File: src/task-manager-client/src/app/models/cron-task.ts
export interface CronTask {
    id: number;
    name: string;
    description: string;
    minutes: string;
    hours: string;
    days: string;
    months: string;
    weekdays: string;
    userId: number;
    apiId: number;
    urlParamsString: string;
}